apps/builddao/widget/components/modals/CreateProject.jsx

```diff
@@ -0,0 +1,304 @@
+const accountId = context.accountId;
+const { Modal, Button, InputField, TextEditor } = VM.require(
+  "buildhub.near/widget/components"
+) || {
+  Modal: () => <></>,
+  Button: () => <></>,
+  InputField: () => <></>,
+  TextEditor: () => <></>,
+};
+
+const showModal = props.showModal;
+const toggleModal = props.toggleModal;
+const toggle = props.toggle;
+
+const tabs = [
+  { id: "overview", label: "Overview", disabled: true, checked: true },
+  { id: "discussion", label: "Discussion", checked: false },
+  { id: "tasks", label: "Tasks", checked: false },
+  { id: "code", label: "Code", checked: false },
+  { id: "roadmap", label: "Roadmap", checked: false },
+];
+
+const [tags, setTags] = useState(props.filters.tags ?? []);
+const [title, setTitle] = useState("");
+const [description, setDescription] = useState("Enter Project Description");
```

Comment:
rather than having it as value, it should be placeholder

Reply:
Absolutely, yes